Source/DafnyLanguageServer/Workspace/ProjectManagerDatabase.cs

```diff
@@ -87,7 +87,8 @@ public class ProjectManagerDatabase : IProjectDatabase, IDisposable {
     public async Task<IdeState?> GetResolvedDocumentAsync(TextDocumentIdentifier documentId) {
       var manager = await GetProjectManager(documentId, false);
       if (manager != null) {
-        return await manager.GetSnapshotAfterResolutionAsync()!;
+        var result = await manager.GetSnapshotAfterResolutionAsync()!;
```

Comment:
nit/non-blocking: why is this needed?

Reply:
Woops, accidentally checked this in. Will remove in a follow-up